### PR TITLE
mkdir: add command to insert multiple directories with example

### PR DESCRIPTION
**Pull request by Nivethan-Ar**
using mkdir folder1 folder2 command we can create multiple directories in the current directory.

- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Review by schneiderl — APPROVED** [review]

**Comment by CleanMachine1** [comment]
@schneiderl I wondered why it gave the option to merge, luckily I saw it. This is being pushed to master rather than main which doesn't have branch protection

**Comment by CleanMachine1** [comment]
@Nivethan-Ar Please always PR to main rather than master.

Master still exists so tldr clients using it don't become outdated.

File: pages/common/mkdir.md
Comment (CleanMachine1) [inline pages/common/mkdir.md:12]:
```suggestion
- Create multiple directories in current directory:

`mkdir {{directory_1 directory_2 ...}}`

```

**Comment by CleanMachine1** [comment]
Why have you closed.

**Comment by Nivethan-Ar** [comment]
this is my first PR on github can you guide me on what to do in order to contribute this change?

**Review by CleanMachine1 — APPROVED** [review]
Thanks for your contributions! 

Before we merge, we wait 24 hours since opening the PR to let all active maintainers have a look before merging.

Once its agreed that everyone approves then it will be merged.

Thanks.

**Comment by Nivethan-Ar** [comment]
thank you @CleanMachine1 for your help. 🙃

File: pages/common/mkdir.md
Comment (marchersimon) [inline pages/common/mkdir.md:10]: We always try to be as verbose as possible:
```suggestion
- Create multiple directories in the current directory:
```

**Review by marchersimon — APPROVED** [review]
Thanks @Nivethan-Ar

**Pull request merged**
